# returns the number of joints of the body
string sessionname
int32 sessionid
uint32 bodyid
---
int32 dof
